client/my-sites/sidebar/style.scss

```diff
@@ -0,0 +1,21 @@
+.sidebar__sparkline {
+	width: 80px;
+	height: 24px;
+	position: absolute;
+	right: 16px;
+	top: 8px;
+
+	@include breakpoint( "<660px" ) {
+		top: 13px;
```

Comment:
Should we consider showing the sparkline below this breakpoint regardless of the stats section being selected?

The background color no longer obscures the image & the sidebar entirely covers the main layout component when visible.

Additionally, switching between sections below this breakpoint creates a sort of popping in of the sparkline: https://cloudup.com/ckMODiimxiD (it's more pronounced on a site with more traffic)